telemetry/specs/INT.mdk

```diff
@@ -755,9 +755,9 @@ and the metadata itself.
 INT Metadata Header and Metadata Stack:
 `
     0                   1                   2                   3
-    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5     6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   |  Ver  |Rep|C|E|M|     Reserved      | Hop ML  |RemainingHopCnt|
+   |  Ver  |Rep|C|E|M|D|    Reserved     | Hop ML  |RemainingHopCnt|
```

Comment:
Do we need the D bit if bit 14 in instruction bitmap conveys the same information?

Reply:
We concluded that the bit is needed to convey the presence of Domain Specific Flags.